app/views/ConfirmationView.scala.html

```diff
@@ -16,28 +16,51 @@
 
 @import config.AppConfig
 @import models.requests.DataRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import java.time.LocalDate
+@import java.time.format.DateTimeFormatter
 
 @this(
         layout: templates.Layout,
         confirmationPanel: components.confirmationPanel,
         p: components.p,
-        strong: components.strong,
         link: components.link,
-        appConfig: AppConfig
+        h2: components.h2,
+        appConfig: AppConfig,
+        govukSummaryList: GovukSummaryList
 )
 
-@(reference: String)(implicit request: DataRequest[_], messages: Messages)
+@(reference: String, dateOfSubmission: LocalDate)(implicit request: DataRequest[_], messages: Messages)
 
 @layout(pageTitle = titleNoForm(messages("confirmation.title")), maybeShowActiveTrader = maybeShowActiveTrader(request), showBackLink = false) {
 
     @confirmationPanel("confirmation.heading")
 
-    @p() {
-        @messages("confirmation.reference")
-        @strong {
-            @reference
-        }
-    }
+    @h2("confirmation.movementInformation")
+
+    @govukSummaryList(SummaryList(
+        rows = Seq(
+            SummaryListRow(
+                key = Key(Text(messages("confirmation.localReferenceNumber"))),
+                value = Value(Text(reference))
+            ),
+            SummaryListRow(
+                key = Key(Text(messages("confirmation.dateOfSubmission"))),
+                value = Value(Text(dateOfSubmission.format(DateTimeFormatter.ofPattern("dd LLLL yyyy"))))
+            )
+        )
+    ))
+
+    <p class="govuk-body no-print"><a class="govuk-link" href="javascript:if(window.print)window.print()">@messages("confirmation.printLink")</a> @messages("confirmation.printMessage")</p>
+
+    @h2("confirmation.whatHappensNext")
+    <p class="govuk-body">@messages("confirmation.p1")</p>
+    <p class="govuk-body">@messages("confirmation.p2.beforeLink") <a class="govuk-link" href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/excise-enquiries"> @messages("confirmation.p2.link")</a> @messages("confirmation.p2.afterLink")</p>
```

Comment:
We've moved external links into config in other services, might be worth doing here too